Foundation/NSURLResponse.swift

```diff
@@ -167,9 +178,9 @@ public class NSHTTPURLResponse : NSURLResponse {
         @result A dictionary containing all the HTTP header fields of the
         receiver.
     */
-    public var allHeaderFields: [NSObject : AnyObject] { NSUnimplemented() }
+    public let allHeaderFields: [String : String]
```

Comment:
This should have a comment on why the differential and identifying it as an "experimental" change; albeit that I agree that this should be the signature, I have not yet heard back from the component owner on a definitive reason on either why it should or should not be marked as string to string.